packages/node_modules/@webex/plugin-meetings/src/media/index.js

```diff
@@ -103,15 +102,16 @@ Media.generateLocalMedias = (mediaId, audioMuted, videoMuted) => {
 /**
  * make a browser call to get the media
  * @param {SendOptions} options
+ * @param {Object} config SDK Configuration for meetings plugin
  * @returns {Promise}
  */
-Media.getLocalMedia = (options) => {
+Media.getLocalMedia = (options, config) => {
```

Comment:
You could add the 'config' object to the 'options' object instead.

Reply:
The media class could stand to be refactored a bit so we don't have to pass the config, but stores it in the class itself. I was hoping having it separate would make it easier to refactor.